modules/test/playwright/tests/dynamic-data-mapping-form-web/lxc-environment/formDataProvider.spec.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import path from 'path';
+
+import {formsPagesTest} from '../../../fixtures/formsPagesTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import {virtualInstancesPagesTest} from '../../../fixtures/virtualInstancesPagesTest';
+import {liferayConfig} from '../../../liferay.config';
+import {FormBuilderPage} from '../../../pages/dynamic-data-mapping-form-web/FormBuilderPage';
+import {FormsPage} from '../../../pages/dynamic-data-mapping-form-web/FormsPage';
+import performLogin, {performLogout} from '../../../utils/performLogin';
+import {deleteItems} from '../utils/deleteItems';
+
+export const test = mergeTests(
+	loginTest(),
+	formsPagesTest,
+	virtualInstancesPagesTest
+);
+
+let hasDataProvider: boolean = false;
+
+const DEFAULT_VIRTUAL_INSTANCE_NAME = 'www.able.com';
+
+test.afterEach(async ({formsPage, page}) => {
+	await formsPage.goTo();
+
+	await deleteItems(formsPage, page);
```

Comment:
This is not part of this PR, but I will add a comment about anyway: (almost) every `ComponentPage` is instantiated with a `new ComponentPage(page)`, so it already has a reference to Playwrights page instance. This `deleteItems` util does not need to receive two arguments. This util could be:

```
import {FormsPage} from '../../../pages/dynamic-data-mapping-form-web/FormsPage';

export async function deleteItems(formsPage: FormsPage) {
	await formsPage.page.waitForTimeout(1000);

	if (formsPage.managementToolbarSelectAllItems.isEnabled()) {
		await formsPage.managementToolbarSelectAllItems.click();

		formsPage.page.once('dialog', (dialog) => {
			dialog.accept();
		});

		await formsPage.managementToolbarDeleteButton.click();
	}
}
```
You don't need to change this now. I have created a [technical debit](https://liferay.atlassian.net/browse/LPD-39111) for this.